### PR TITLE
Create common troubleshoot errors page for Autopilot

### PR DESCRIPTION
**Pull request by joa-pan**
Pull requests should be opened in a branch off the `main` branch. For more information on contributing to Pantheon documentation, see the [Contributor Guidelines](https://pantheon.io/docs/contribute). Please refer to our [Style Guide](https://pantheon.io/docs/style-guide) and [this reference](https://developers.google.com/style) for formatting recommendations when contributing to the docs. 

Closes #

## Summary

**[Autopilot Errors](https://pantheon.io/docs/guides/autopilot)** -  Added a page for common Autopilot errors and troubleshooting solutions. 

## Effect

The following changes are already committed:

*
*

## Remaining Work

The following changes still need to be completed:

- [ ] List any outstanding work here

--------------------------------------------------
## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Redirect `/docs/old-path/` => `/docs/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] For Heroes - add a props post to the [discussion board](https://discuss.pantheon.io/c/pantheon-platform/documentation/17).
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by joa-pan** [comment]
@EdwardAngert When you have a moment, can you please review?

File: source/content/guides/autopilot/04-troubleshoot.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+```
+
+After that workflow completes, refresh the Autopilot settings or restart the initialization wizard. The extension options should be present.
+
+
+##  Drush Version
+Error Message: "Drush version 8 required" 
+
+### Issue
+Autopilot only works on Drupal sites that are running Drush 8. Drush 5/7/9/10 are not supported.
```
Comment (greg-1-anderson) [inline source/content/guides/autopilot/04-troubleshoot.md:46]: Note that this limitation does not apply to sites that are Composer-managed. Composer-managed sites should use Drush 10; they should not receive the Drush version error.

Note that currently, Autopilot only supports Integrated Composer; build tools sites cannot be updated.

File: source/content/guides/autopilot/04-troubleshoot.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+### Issue
+
+Since the Autopilot plan is only set after Autopilot is initialized, the list of elements is unavailable when Autopilot is started for the first time.
+
+When Autopilot is started for the first time or the Slow Converge feature is used, refresh the extensions list used for exclusions.
+
+### Solution
+
+Trigger a workflow to refresh extensions from the GraphQL Playground with the following code change:
```
Comment (EdwardAngert) [inline source/content/guides/autopilot/04-troubleshoot.md:30]: can you give users a quick line about how they can access the GraphQL Playground or point to other docs? I thought this was internal only

File: source/content/guides/autopilot/04-troubleshoot.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+### Diagnosis
+
+To determine which plugin is causing the issue:
+
+1. Check the uncommitted changes for Autopilot Multidev. It is likely a plugin has been deleted and another plugin has been added.
+
+1. Compare the list of plugins for Autopilot and Dev to the WordPress plugin list and determine if there are any discrepancies. Ensure no entries are missing. If there is variance among the lists, such as a plugin that has been omitted or not updated on one of the lists, the plugin will be disabled on the Autopilot branch.
+
+Contact an Autopilot engineer to check the state machine logs, which requires AWS access.
```
Comment (EdwardAngert) [inline source/content/guides/autopilot/04-troubleshoot.md:75]: not sure how users would contact an Autopilot engineer. Should we just have them contact Support?

File: source/content/guides/autopilot/04-troubleshoot.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+This page is a list of common issues and solutions that you may encounter when using Autopilot.
+
+## Error: Element Exclusions
+
+> We didn’t detect any plugins, modules or themes that were eligible for exclusion.
+
+### Issue
+
+Since the Autopilot plan is only set after Autopilot is initialized, the list of elements is unavailable when Autopilot is started for the first time.
```
Comment (EdwardAngert) [inline source/content/guides/autopilot/04-troubleshoot.md:24]: I want to reword the `Autopilot plan` part, but I'm not sure what to. Maybe about when the Autopilot feature is initialized? "Plan" sounds more internal.

File: source/content/guides/autopilot/04-troubleshoot.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+
+When Autopilot is started for the first time or the Slow Converge feature is used, refresh the extensions list used for exclusions.
```
Comment (EdwardAngert) [inline source/content/guides/autopilot/04-troubleshoot.md:26]: Slow Converge? isn't that a CSE only thing?

**Review by EdwardAngert — CHANGES_REQUESTED** [review]

**Pull request merged**